flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -466,4 +467,8 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
       )
     result.success(serializedScope)
   }
+
+  private fun crash(result: Result) {
+    error("This is a fatal error caused by SentryFlutter.nativeCrash.")
```

Comment:
@buenaflor @denrase can you please have a look at this implementation? Feel free to improve, as I am not a Kotlin expert.

Reply:
I improved the implementation. So this should be sufficient now.